FearlessUtils/Classes/Scale/DynamicScaleCoding+Codable.swift

```diff
@@ -3,21 +3,31 @@ import Foundation
 public class HexCodingStrategy {
     static func encoding(data: Data, encoder: Encoder) throws {
         var container = encoder.singleValueContainer()
-        let hex = data.toHex(includePrefix: true)
-        try container.encode(hex)
+        let array = data.map { String($0) }
```

Comment:
Looks like it's not "hex" coding strategy anymore. Just confusing class naming, imho